src/components/KeyboardShortcutsModal.js

```diff
@@ -35,52 +35,69 @@ const defaultProps = {
 class KeyboardShortcutsModal extends React.Component {
     componentDidMount() {
         const openShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
+        this.subscribedOpenModalShortcuts = [];
         this.unsubscribeShortcutModal = KeyboardShortcut.subscribe(openShortcutModalConfig.shortcutKey, () => {
             ModalActions.close();
             KeyboardShortcutsActions.showKeyboardShortcutModal();
         }, openShortcutModalConfig.descriptionKey, openShortcutModalConfig.modifiers, true);
 
+        if (this.props.isShortcutsModalOpen) {
+            // The modal started open, which can happen if you reload the page when the modal is open.
+            this.subscribeOpenModalKeyboardShortcuts();
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (!prevProps.isShortcutsModalOpen && this.props.isShortcutsModalOpen) {
+            this.subscribeOpenModalKeyboardShortcuts();
+        } else if (prevProps.isShortcutsModalOpen && !this.props.isShortcutsModalOpen) {
+            // Modal is closing, remove keyboard shortcuts
+            this.unsubscribeOpenModalShortcuts();
+        }
+    }
+
+    componentWillUnmount() {
+        if (this.unsubscribeShortcutModal) {
+            this.unsubscribeShortcutModal();
+        }
+        this.unsubscribeOpenModalShortcuts();
+    }
+
+    /*
+     * Subscribe shortcuts that only are used when the modal is open
+     */
+    subscribeOpenModalKeyboardShortcuts() {
```

Comment:
NAB: Change function name to `subscribeOpenModalShortcuts` (or change the other one)

Reply:
I remove the `Keyboard` word from the name because it wasn't there in other places.